nservicebus/hosting/cloud-services-host/index.md

```diff
@@ -21,145 +21,45 @@ The Azure Platform and NServiceBus make a perfect fit. On the one hand the Azure
 
-Usually NServiceBus is configured as a client in the IIS process. This needs to be approached in the same way as any other website, by means of self hosting. When self-hosting everything can be configure using the configuration API and the extension methods found in the `NServiceBus.Azure` package. No reference to the hosting package is required in that case.
+Usually NServiceBus is configured as a client in the IIS process. This needs to be approached in the same way as any other website, by means of self hosting. When self-hosting everything can be configure using the configuration API and the extension methods found in the nservicebus azure related packages. No reference to the hosting package is required in that case.
 
-The configuration API is used with the following extension methods to achieve the same behavior as the generic `AsA_worker`:
+The configuration API is used with the following extension methods to achieve the same behavior as the `AsA_worker` role:
 
 snippet:HostingInWebRole
 
 A short explanation of each:
 
  * `AzureConfigurationSource`: overrides any settings known to the NServiceBus Azure configuration section within the app.config file with settings from the service configuration file.
- * `TraceLogger`: Redirects all [logging](/nservicebus/logging/) to the trace logger. This in turn can be configured for diagnostics monitor trace listener.
+ *  Logs will be sent to the `Trace` infrastructure, which should have been configured with Azure diagnostic monitor trace listener by the visual studio tooling.
  * `UseTransport<AzureStorageQueueTransport>`: Sets [Azure storage queues](/nservicebus/azure-storage-queues/) as the [transport](/nservicebus/transports).
- * `UsePersistence`: Configures [Azure storage](/nservicebus/azure-storage-persistence/) for [persistence](/nservicebus/persistence).
-
-
-include:host-startup
-
-
-## Handling critical errors
-
-
-### Azure Host Versions 6.2.2 and above
-
-Azure host is terminated on critical errors by default. When host is terminated, Azure Fabric will restart the host automatically.
-
-
-### Azure Host Versions 6.2.1 and below
-
-Azure host is not terminated on critical errors by default and only shuts down the bus. This would cause role not to process messages until host (role) is restarted. To address this, implement critical errors handling code that shuts down the host.
+ * `UsePersistence<AzureStoragePersistence>`: Configures [Azure storage](/nservicebus/azure-storage-persistence/) for [persistence](/nservicebus/persistence).
 
-snippet:DefineCriticalErrorActionForAzureHost
\ No newline at end of file
+For more information on configuration refer to the [configuration page](configuration.md). For more information on setting up logging, refer tot the [logging page](logging.md) 
```

Comment:
> configuration page

[Azure Cloud Services Host Endpoint Configuration](link) page ?

> logging page

[Logging in NServiceBus](link) page?